bin/add_users.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import os
+
+# Add the parent directory to sys.path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+
+from app.db import get_session
+from app.models.users import Users
+from app.auth.security import get_password_hash
+import logging
+
+
+def add_users(users_list_dict: dict):
+    """
+    This function creates new users in the spun up local database
+
+    Args:
+        users_list_dict: Should contain a list of dictionary
+        usernames and plain text passwords for users
```

Comment:
What data type is `users_list_dict` supposed to be? A list or a dictionary, I think the data type in the function definition should be `def add_users(users_list_dict: list[dict]):`